peachjam/js/components/DocDiffs/ProvisionChangedGutterItem.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <la-gutter-item
+    :anchor="`#${provision.id}`"
+    class="d-none d-lg-block"
+  >
+    <div class="card">
+      <div class="card-body p-3">
+        <p>{{ $t('This provision has been amended') }}.</p>
+        <button
+          class="btn btn-sm btn-secondary"
+          type="button"
+          @click="showChanges"
+        >
+          {{ $t('What changed') }}?
```

Comment:
```suggestion
          {{ $t('What changed?') }}
```

Questions in some languages change word order, so we need the ? to indicate to the translator that it's a question.